src/app/app.component.scss

```diff
@@ -1 +1,48 @@
+@import 'src/app/styles/layout/variable.scss';
+/* Adjust logo size for smaller screens */
+.sidenav {
+  background: $background-color ;
+  overflow: hidden;
+}
+.navbar-brand {
+  font-size: 32px; /* changed from 2rem to 32px */
+  margin-bottom: 0;
+  color: #ffff;

```

Comment:
please check guideline to use color code. #ffff -> #fff

Reply:
change done.